x-pack/platform/packages/shared/kbn-elastic-assistant/impl/knowledge_base/knowledge_base_settings_management/document_entry_editor.tsx

```diff
@@ -49,6 +49,7 @@ export const DocumentEntryEditor: React.FC<Props> = React.memo(
         setEntry((prevEntry) => ({
           ...prevEntry,
           users: value === i18n.SHARING_GLOBAL_OPTION_LABEL ? [] : privateUsers,
+          global: value === i18n.SHARING_GLOBAL_OPTION_LABEL ? true : false,
```

Comment:
Is this a UI fix follow-up for `global` that was added in https://github.com/elastic/kibana/pull/211367? If so, will this need to be in `8.18`?

Reply:
correct